templates/edit.jinja2

```diff
@@ -0,0 +1,35 @@
+{% extends "base.jinja2" %}
+{% block body %}
+{% if request.authenticated_userid %}
+<h2>Edit</h2>
+<aside>
+<form action="{{ request.route_url('add') }}" method="POST" class="add_entry">
+  <div class="field">
+    <label for="title">Title</label>
+    <input type="text" size="30" name="title" id="title"/>
+  </div>
+  <div class="field">
+    <label for="text">Text</label>
+    <textarea name="text" id="text" rows="5" cols="80"></textarea>
+  </div>
+  <div class="control_row">
+    <input type="submit" value="Share" name="Share"/>
+  </div>
+</form>
+</aside>
+{% endif %}
+<h2>Entries</h2>
+  {% for entry in entries %}
+  <article class="entry" id="entry={{entry.id}}">
+    <h3>{{ entry.title }}</h3>
+    <p class="dateline">{{ entry.date.strftime('%b. %d, %Y') }}
+    <div class="entry_body">
+      {{ entry.text|safe }}
+    </div>
+  </article>
+  {% else %}
+  <div class="entry">
+    <p><em>No entries here so far</em></p>
+  </div>
+  {% endfor %}
+{% endblock %}
```

Comment:
Not sure about the purpose of this template as opposed to all the others with the same content.